Watchman.Integrations/Disboard/ServerBumper.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Discord.WebSocket;
+
+namespace Watchman.Integrations.Disboard
+{
+    public class ServerBumper
+    {
+        private const string BumpMessage = "!d bump";
```

Comment:
dlaczego z dużej litery skoro private?

teoretycznie const powinien być napisany w całości dużymi literami
przykładowo BUMP_MESSAGE 
tak, wiem że w którymś z miejsc zrobiłem to źle

możemy przyjąć że const traktujemy jak normalne fieldy przy nazewnictwie